doc/faq.rst

```diff
@@ -126,7 +126,7 @@ Running Spyder
 Using Spyder
 ============
 
-.. dropdown:: Q: How do I install Python packages to use within Spyder?
+.. dropdown:: Q: How do I install Python packages to use within Spyder if installed Spyder with conda?
```

Comment:
I'd rewrite this as "How do I install Python packages to use within Spyder if I installed Spyder with conda?" (with "I" added after "if") for grammatical correctness.

Reply:
Sounds good to me.